app/src/main/res/layout/activity_get_pet.xml

```diff
@@ -67,13 +67,32 @@
             android:id="@+id/shelter_name"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginBottom="8dp"
+            android:layout_marginBottom="16dp"
             android:fontFamily="@font/roboto_medium"
             android:gravity="center"
+            android:textStyle="bold"
             android:textColor="@color/colorAccent"
             android:textSize="@dimen/text_medium"
             tools:text="VsI Naminukas" />
 
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="16dp"
+            android:fontFamily="@font/roboto"
+            android:gravity="center"
+            android:textColor="@color/colorAccent"
+            android:textSize="@dimen/text_medium"
+            android:text="@string/push_and_contact"/>
+
+        <ImageView
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_marginBottom="16dp"
+            android:layout_gravity="center"
+            android:scaleType="centerInside"
+            android:src="@drawable/double_down" />
```

Comment:
Let's use `app:srcCompat` with vector drawables. Fore more information why https://medium.com/androiddevelopers/appcompat-v23-2-age-of-the-vectors-91cbafa87c88